src/kata2.booleanToWord.js

```diff
@@ -1,3 +1,9 @@
-const booleanToWord = boolean => {};
-
+const booleanToWord = (boolean) => {
```

Comment:
Brackets can be omitted as before
```suggestion
const booleanToWord = boolean => {
```